flux.helm-controller.yaml

```diff
@@ -0,0 +1,29 @@
+package:
+  name: flux-helm-controller
```

Comment:
Nit - I think the file names have to match the package name (so flux-helm-controller, instead of flux.helm-controller)

Reply:
ack, was wondering whether that would break things...